docs/index.md

```diff
@@ -4,7 +4,7 @@ The Mapzen terrain tiles provide basemap elevation coverage of the world in a ra
 
 Our raster tiles are square-shaped grids of geographic data that contain elevation data in a raw or processed format.
 
-Tiles are available for zooms 0 through 15 and are available in several spatial data formats including processed normal PNGs, which are optimized for mobile and web display, and raw elevation GeoTIFFs for desktop analytical use. Data is available in both projected Web Mercator (EPSG:3857) and raw latlng formats.
+Tiles are available for zooms 0 through 15 and are available in several spatial data formats including processed PNG images, which are optimized for mobile and web display, and raw elevation GeoTIFFs for desktop analytical use. Data is available in both projected Web Mercator (EPSG:3857) and non-projected geographic coordinate system [WGS84](http://spatialreference.org/ref/epsg/wgs-84/) (EPSG:4326) formats.
```

Comment:
Option: Alternatively link to Wikipedia's https://en.wikipedia.org/wiki/World_Geodetic_System on WGS84, and then move the spatial ref link to 4326?

Otherwise LGTM and can be merged.